src/exporter/wp_exporter.py

```diff
@@ -368,6 +369,17 @@ def import_pages(self):
                     'post_status': 'publish',
                 }
 
+            # If the page doesn't exist for all languages on the site we create a blank page in draft status
+            # At the end of export we delete all draft pages
+            for lang in self.wp_generator._site_params['langs'].split(","):
+                if lang in info_page:
```

Comment:
Tu pourrais t'épargner les 2 lignes du `if` et `continue` en mettant simplement un check sur `if lang not in info_page:` suivi de ton code. 
D'autant plus que là, ton `else:` après le `continue` ne sert à rien 😋 tu pourrais simplement mettre ton code sans celui-ci 😉

Reply:
Je suis ok je corrige